pcs/cli/resource/parse_args.py

```diff
@@ -50,16 +50,15 @@ def parse_clone(arg_list, promotable=False):
         raise CmdLineInputError(
             "op settings must be changed on base resource, not the clone",
         )
-    parts["meta"] = prepare_options(
-        groups.get("options", []) + groups.get("meta", []),
-    )
+
+    if "meta" in groups:
+        raise CmdLineInputError(
+            "meta options is not supported in 'pcs resource clone' command"
+        )
```

Comment:
For now, we need to allow `meta` keyword to maintain backward compatibility. So we cannot raise an exception here. Instead, print a deprecation warning using `pcs.cli.reports.output.deprecation_warning`.